<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowsWellbeing</class>
 <widget class="QMainWindow" name="WindowsWellbeing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window Wellbeing</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 231, 231);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="todayBt">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
     </property>
     <property name="text">
      <string>Today's stats</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prevBt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
     </property>
     <property name="text">
      <string>Previous Stats</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutBt">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>1191</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QFrame" name="todayOption">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>460</y>
        <width>801</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="foregroundBt">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
       </property>
       <property name="text">
        <string>Foreground apps</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backgroundBt">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>0</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
       </property>
       <property name="text">
        <string>Background apps</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1171</width>
        <height>461</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QStackedWidget" name="foreAndBackStackedToday">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>791</width>
         <height>451</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="foreground">
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>0</y>
           <width>801</width>
           <height>461</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="todayFrQhbox"/>
        </widget>
       </widget>
       <widget class="QWidget" name="background">
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>0</y>
           <width>801</width>
           <height>471</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="backgroundQhbox"/>
        </widget>
       </widget>
       <widget class="QWidget" name="noTodayDataPage">
        <widget class="QLabel" name="noDataLabelToday">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>160</y>
           <width>351</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>No statistics found for today. Reboot your computer or try running script manually</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>791</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QStackedWidget" name="foreAndBackStackedPrev">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>781</width>
         <height>451</height>
        </rect>
       </property>
       <widget class="QWidget" name="foreground_prev">
        <widget class="QWidget" name="horizontalLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>781</width>
           <height>441</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="prevFrQhbox"/>
        </widget>
       </widget>
       <widget class="QWidget" name="background_prev">
        <widget class="QWidget" name="horizontalLayoutWidget_8">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>781</width>
           <height>441</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="prevBkQhbox"/>
        </widget>
       </widget>
       <widget class="QWidget" name="no_prev_data">
        <widget class="QLabel" name="no_data_prev">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>160</y>
           <width>161</width>
           <height>121</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Sans Serif</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>No data found</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="prev_option">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>781</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="foregroundBt_prev">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
       </property>
       <property name="text">
        <string>Foreground apps</string>
       </property>
      </widget>
      <widget class="QPushButton" name="backgroundBt_prev">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:2px solid white;
	background-color:white;
	outline:none;
}
QPushButton:focus{
	border:2px solid grey;
}</string>
       </property>
       <property name="text">
        <string>Background apps</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="date_select">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>10</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QLabel" name="thanks">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Thanks for trying out this software </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="link">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Link to github source code &lt;a href=&quot;https://github.com/pratik0903/Windows-Wellbeing&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;here&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Windows-Wellbeing/qtImage.qrc"/>
 </resources>
 <connections/>
</ui>
